### DISCUSSION TITLE
Did way of calculating knps change?

### DISCUSSION THREAD

**Discussion by Kicer86** [lichess-org/fishnet#256] [Q&A] [answered]
Hi

I used to observe ~15000knps on PC for months, but recently (lets say 2-3 months) it is around 5000knps.
I didn't change fishnet's configuration nor altered my hardware. 
Has anything related to this changed in fishnet? Or is my cpu protesting?

**Comment by niklasf** [comment, marked as answer]
Yes, in [v2.8.x](https://github.com/lichess-org/fishnet/releases/tag/v2.8.2). First, slightly:

> Compared to v2.7.x, 20% lower nodes per wall clock second with significantly increased variance are expected, but quality is improved.

And then if your CPU has a large number of cores (which I guess it has?), in a major way. Or rather: The counting has *not* changed. It was always counting the nodes per second for the particular game/batch. But due to the new chunking, not all cores may be able to work on a single game.

**Reply by Kicer86** [reply]
Oh I see, thank you for explanations. Yes I use many cores for fishnet: 12.
